doc/source/whatsnew/v0.23.0.rst

```diff
@@ -1,7 +1,13 @@
 .. _whatsnew_0230:
 
-v0.23.0 (May 15, 2018)
-----------------------
+
+********************
+What's New in 0.23.0
+********************
+
+{{ common_imports }}
+
+**v0.23.0 (May 15, 2018)**
```

Comment:
Changing the header formatting from `v.0.23` is very likely intended, but just pointing out that it is only used for `0.23.*` and not before or after. (also: not sure I'm the biggest fan with the redundancy of having the version twice)

Reply:
This header `**v0.23.0 (May 15, 2018)**` is still redundant